open-company/open-company-web
# Static site + Boot

trello: https://trello.com/c/M61wEIwH/360-static-site-generator-for-public-website

### Contents:

1. add a static site that resembles what is currently on opencompany.com
2. move cljs compilation/testing and sass compilation to boot
3. setup ansible/infrastructure stuff to use new static site and boot based tooling
4. rename all tests to match namespace <> directory structure 

### To Test

1. build everything locally and play around with reloading (sass/cljs) using `boot dev`
2. build app for production using `boot prod-build`
3. deploy to staging and verify the site is identical to opencompany.com & the app still works

---

- [x] delete `project.clj`
- [x] delete devcards related stuff
- [x] update `prod-build` task to use `externs.js`
